OrbitQt/OrbitStartupWindow.h

```diff
@@ -50,11 +50,10 @@ class OrbitStartupWindow : public QDialog {
     if (std::holds_alternative<SshInfo>(result_)) {
       auto& ssh_info = std::get<SshInfo>(result_);
       Credentials credentials{};
-      credentials.host = ssh_info.host.toStdString();
```

Comment:
Commit message - second line: Capitalize Refactored
last line: google C++ style guilde

Reply:
I guess commit message changing means a force push is necessary. Does gerrit also solve that? ;)